admin/views/tool-file-editor.php

```diff
@@ -82,19 +82,19 @@
 $helpcenter->localize_data();
 $helpcenter->mount();
 
-echo '<h2>', __( 'Robots.txt', 'wordpress-seo' ), '</h2>';
+echo '<h2>', esc_html__( 'Robots.txt', 'wordpress-seo' ), '</h2>';
```

Comment:
I know this is not the scope of this pull, but robots.txt shouldn't be translatable?

Reply:
Done.